docs/index.html

```diff
@@ -51,5 +51,7 @@
         countIncrementalSlides: false,
       });
     </script>
+    <!-- Required to avoid Chromiium's lazy-loading of fonts -->
+    <div class="space-mono">a</div>
```

Comment:
This can be fixed I think by dropping the `display` parameter in the places we load fonts.

Reply:
Tried it, didn't work